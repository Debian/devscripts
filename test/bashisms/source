#!/bin/sh
source foo.sh
