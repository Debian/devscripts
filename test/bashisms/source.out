possible bashism in bashisms/source line 2 (should be '.', not 'source'):
source foo.sh # BASHISM
